CarLot.js

```diff
@@ -0,0 +1,40 @@
+"use strict"
+
+  let inventory = [];
+  let carCards = document.getElementsByClassName("carCard");
+
+let CarLot = (function () {
+
+
+  return {
+    getInventory: function () {
+      return inventory;
+
+    },
+    loadInventory: function () {
```

Comment:
In your `quiz.js` you are calling this function and you are passing your callback into it, but you don't have your callback defined in the actual `loadInventory` function.